justfile

```diff
@@ -7,4 +7,9 @@ install-contracts:
   forge install
 
 install-eip712sign:
-  go install github.com/base-org/eip712sign@v0.0.3
+  #!/usr/bin/env bash
+  cd ~
+  git clone https://github.com/ethereum-optimism/eip712sign.git
```

Comment:
`just install` is not idempotent and not intended to be executed multiple times

but we can clean up the old repo before cloning, so every time the user executes `just install` it will fetch the latest commit from the repo